src/js/starting-page.js

```diff
@@ -0,0 +1,23 @@
+const startingPage = (renderFunction, HtPRenderFunction) => {
+    const title = document.createElement("h1");
+    title.id = "title";
+    title.textContent = "Unnamed Game";
+
+    const desc = document.createElement("p");
+    desc.id = "description";
+    desc.textContent = "Lorem ipsum dolor sit amet, consectetur adipiscing elit. In gravida consectetur est at pharetra. Cras quis arcu convallis magna egestas feugiat."
+
+    const startGameBtn = document.createElement("button");
+    startGameBtn.id = "start-game-button";
+    startGameBtn.textContent = "Start game";
+    startGameBtn.addEventListener("click", renderFunction);
+
+    const HtPBtn = document.createElement("button");
+    HtPBtn.id = "how-to-play-button";
+    HtPBtn.textContent = "How to play";
+    HtPBtn.addEventListener("click", HtPRenderFunction);
+
+    return { title, desc, startGameBtn, HtPBtn }
```

Comment:
From our discussion we've had so far, maybe it's a bit more practical to append all elements to a main node like you did before and export this main node. 

```
const mainNode = document.createElement("div");
mainNode.append (title)
mainNode.append (textContent)
and so on
return { mainNode }
```
But I'm not so sure, it depends a bit on what we come up with to tie all component together.

Reply:
Okay! I can do that, and if we need to change it later it's not much to change.

Reply:
^ I agree. This way we only have to pass one thing into a render function for it to be displayed